superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -332,6 +333,38 @@ const thumbnailContainerCss = (theme: SupersetTheme) => css`
   }
 `;
 
+const BetaBadge = styled.div`
+  ${({ theme }) => `
+    border: 1px solid #1985a0;
+    box-sizing: border-box;
+    border-radius: 4px;
+    background: #ffffff;
+    line-height: ${theme.gridUnit * 2.5}px;
+    color: #1985a0;
+    font-size: 12px;
+    font-weight: 500;
```

Comment:
```suggestion
    font-weight: ${theme.typography.weights.bold};
```
This will put it at 700. Instead of `bold` we could use `normal` which is 400. I figure this kind of label is usually bold, thus my guess at the suggestion here.

@kasiazjc your call if we want to (a) pick 400 or 700, (b) add more weights to the theme, or (c) make this a special exception.